elem/elem.gen.go

```diff
@@ -754,7 +754,7 @@ func Small(markup ...vecty.Markup) *vecty.Element {
 	return e
 }
 
-// The HTML <source> element specifies multiple media resources for either the <picture>, the <audio> or the <video> element. It is an empty element. It is commonly used to serve the same media content in multiple formats supported by different browsers.
+// Editorial review completed.
```

Comment:
Shame, the `<source>` tag still has a description of `Editorial review completed.`.

It seems like the primary page won't rebuild for some reason.